odilo.py

```diff
@@ -454,7 +476,7 @@ def record_info_to_metadata(cls, book, availability):
             except ValueError as e:
                 cls.log.warn('Cannot parse last update date from: ' + last_update + ', message: ' + e.message)
 
-        language = book.get('language')
+        language = book.get('language', 'spa')
```

Comment:
Nearly all Odilo books are in Spanish, so this is the Odilo equivalent of assuming an Overdrive book is in English -- not guaranteed to be right but a good guess.